benchmarks/automated/tests/suite/wf_nmt_retranslation.yaml

```diff
@@ -0,0 +1,31 @@
+---
+nmt_re_translation:
+    workflow:
+        benchmark_engine: "ab"
+        workflow_name: "nmt_wf_re"
+        models:
+          - "https://torchserve.pytorch.org/mar_files/TransformerDe2En.mar"
+          - "https://torchserve.pytorch.org/mar_files/TransformerEn2De.mar"
+          - "https://torchserve.pytorch.org/mar_files/TransformerEn2Fr.mar"
+        specfile: "https://raw.githubusercontent.com/pytorch/serve/master/examples/Workflows/nmt_transformers_pipeline/nmt_workflow_retranslation.yaml"
+        workflow_handler: "https://raw.githubusercontent.com/pytorch/serve/master/examples/Workflows/nmt_transformers_pipeline/nmt_workflow_handler_retranslation.py"
```

Comment:
Why need specfile and handler? workflow war should be file or url

Reply:
The automated benchmark loops through different 'batch sizes', in case of a workflow, the batch-size is specified in the config yml within the workflow archive, and cannot be modified using REST APIs as of now, hence we need to create/re-create the archive for different executions of batch sizes.